src/ucm/util/sys.c

```diff
@@ -18,22 +18,38 @@
 
 size_t ucm_get_shm_seg_size(const void *shmaddr)
 {
+    size_t    seg_size  = 0;
+    const int max_retry = 5;
```

Comment:
not sure this is right solution, how many retries would be enough?
what is exactly  the issue?
can we use /proc/$$/smaps instead?